store/models/log_events.go

```diff
@@ -306,27 +320,52 @@ func (le RunLogEvent) Requester() common.Address {
 	return common.BytesToAddress(le.Log.Data[:requesterSize])
 }
 
+// InitiatorRun returns an InitiatorRun instance with all parameters
+// from a run log topic, like RequestID.
+func (le RunLogEvent) InitiatorRun() (InitiatorRun, error) {
+	parser, err := parserFromLog(le.Log)
+	if err != nil {
+		return InitiatorRun{}, err
+	}
+
+	return InitiatorRun{
+		RequestID: null.StringFrom(parser.parseRequestID(le.Log)),
```

Comment:
Should this error if the string is empty?

Reply:
I don't think so, but I'm not 100% sure. It could be a valid request id of 0 from the requester's perspective.